charts/enterprise-logs/environments/default/main.jsonnet

```diff
@@ -0,0 +1,8 @@
+local tanka = import 'github.com/grafana/jsonnet-libs/tanka-util/main.libsonnet';
```

Comment:
For now, I'd consider removing all the Tanka environments and the entire environments directory. See my comment about the `exports` target in the Makefile.